lib/build-cron.js

```diff
@@ -22,8 +23,11 @@ module.exports = function buildCronJobs(cronConfigsDir, collectionName, logClean
 
   if (!cronConfigsDir || !collectionName) {
     const err = new VError('Error: \'cronConfigsDir\' and \'collectionName\' are required params.');
-    raygun.send(err);
-    throw err;
+    return Promise.resolve(raygun.send(err))
```

Comment:
That's what I originally had, and I figured that would work.  But because that of code is run synchronously, the `throw err` ends the node process before the `raygun.send(err)` has resolved, resulting in the error not showing up in Raygun